### PR TITLE
Clarify the order in which AI units attack

### PR DESCRIPTION
**Pull request by DarthGandalf**
I dropped the special case where if non-Ranged units can attack several units of the same tier, the player can choose which one to attack regardless of distance. Needs clarification from devs. Upd: Alex confirmed this.
en-both-38

### CODE REVIEW

File: sections/ai_rules.tex [hunk 1]
```diff
@@ -69,12 +69,14 @@ \subsection*{Combat against AI}
...
-If this is impossible, they attack the closest Unit, prioritizing the lowest tier one.\par
-Ranged \includesvg[height=10px]{\svgs/unit_ranged.svg} Units prioritize attacking other Ranged \includesvg[height=10px]{\svgs/unit_ranged.svg} Units of the same tier, then lower tier, and finally higher tier.
-If there are no Ranged \includesvg[height=10px]{\svgs/unit_ranged.svg} Units for them to target, they prioritize Ground \includesvg[height=10px]{\svgs/unit_ground.svg} and Flying \includesvg[height=10px]{\svgs/unit_flying.svg} Units in the same tier order.
-If there's more than one valid target, they attack the closest one.\par
-If there's ever a tie between equally valid targets for any Units, the player chooses which Unit is attacked.
+Enemy Ground \includesvg[height=10px]{\svgs/unit_ground.svg} and Flying \includesvg[height=10px]{\svgs/unit_flying.svg} Units prioritize attacking Units of the \textbf{same} tier.
+If this is impossible, they attack the Unit of a lower tier (in tier \textbf{descending} order, down to bronze), and if that is also impossible, they attack the Unit of a higher tier (in tier \textbf{ascending} order).\par
```
Comment (qwrtln) [inline sections/ai_rules.tex:73]: How is this sentence improving the existing text? It doubles the sentence length, making it harder to understand. Can you come up with an example where it actually helps?

Comment (qwrtln) [reply sections/ai_rules.tex:73]: Do you mean that gold shooter prioritizes silver target over a bronze one?

Comment (DarthGandalf) [reply sections/ai_rules.tex:73]: Precisely (as long as both these targets are ranged, or if there are no ranged targets at all)

For gold shooter the order is: gold ranged, silver ranged, bronze ranged, azure ranged, gold non-ranged, silver non-ranged, bronze non-ranged, azure non-ranged.

Yes, it makes the text longer, but this clarification was missing.

See todays discussion in discord

File: sections/ai_rules.tex [hunk 2]
```diff
@@ -69,12 +69,14 @@ \subsection*{Combat against AI}
...
+Ranged \includesvg[height=10px]{\svgs/unit_ranged.svg} Units prioritize attacking other Ranged \includesvg[height=10px]{\svgs/unit_ranged.svg} Units of the same tier, then lower tier, and finally higher tier, using the same tier order as above.
+If there are no Ranged \includesvg[height=10px]{\svgs/unit_ranged.svg} Units for them to target, they prioritize Ground \includesvg[height=10px]{\svgs/unit_ground.svg} and Flying \includesvg[height=10px]{\svgs/unit_flying.svg} Units in the same tier order.\par
+
+If there's more than one valid target, they attack the closest one.
```
Comment (DarthGandalf) [inline sections/ai_rules.tex:77]:
```suggestion
In both cases, if there's more than one valid target, they attack the closest one.
```

File: sections/ai_rules.tex (hunk 1 above)
Comment (Legrecofr) [reply sections/ai_rules.tex:73]: Is It possible to add that as an example with \textit {}
For gold shooter the order is: gold ranged, silver ranged, bronze ranged, azure ranged, gold non-ranged, silver non-ranged, bronze non-ranged, azure non-ranged.

Comment (DarthGandalf) [reply sections/ai_rules.tex:73]: different symbol meaning?

**Review by qwrtln — APPROVED** [review]

File: sections/ai_rules.tex (hunk 1 above)
Comment (Legrecofr) [reply sections/ai_rules.tex:73]: I mean the colored stars and the symbol for archer, ground and flying units. Sorry for the bad english.

**Comment by DarthGandalf** [comment]
@Legrecofr like this?
en-both-38

**Comment by Legrecofr** [comment]
Yes! Maybe using - instead of "then"?

**Review by Heegu-sama — APPROVED** [review]

**Pull request merged**